src/couch_mrview/src/couch_mrview_cleanup.erl

```diff
@@ -14,29 +14,37 @@
 
 -export([
     run/1,
-    cleanup_purges/3,
-    cleanup_indices/2
+    cleanup/2
 ]).
 
--include_lib("couch/include/couch_db.hrl").
-
 run(Db) ->
     Indices = couch_mrview_util:get_index_files(Db),
     Checkpoints = couch_mrview_util:get_purge_checkpoints(Db),
     {ok, Db1} = couch_db:reopen(Db),
     Sigs = couch_mrview_util:get_signatures(Db1),
-    ok = cleanup_purges(Db1, Sigs, Checkpoints),
+    ok = cleanup_purges(Db, Sigs, Checkpoints),
```

Comment:
seem odd not to use Db1 after the reopen. explain?

Reply:
Good catch, should be Db1